src/web/components/CookieBanner.tsx

```diff
@@ -121,12 +121,12 @@ export class CookieBanner extends Component<{}, { show: boolean }> {
         this.setState({ show: false });
     };
 
-    public componentDidMount() {
-        const seenBanner = getCookie(consentCookie);
-        if (!seenBanner) {
-            this.setState({ show: true });
-        }
-    }
+    // public componentDidMount() {
```

Comment:
Just in case this temporary change turns into a longer one (happens 🤷‍♂), can we add a comment here to help future devs understand why it was commented out? Maybe with a link to a ticket/PR

Reply:
Good shout. Added comments to both files.